### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by AbdelrhmanAmin**
Added features:
- Add preview image to README.md
- Deploy to Heroku final version.
- Add structure
- Add fontawesome
- Add bootstrap
- Applying the provided design specs to build a UI for the bookstore.

### CODE REVIEW

File: src/components/CategoryFilter.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/prop-types */
+// import PropTypes from 'prop-types';
+// import { useSelector } from 'react-redux';
```
Comment (krys2fa) [inline src/components/CategoryFilter.js]: - [x] [REQUIRED] It’s better to delete **commented piece of codes** like these. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: src/components/CategoryFilter.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          </option>
+        ))}
+      </select>
+    </form>
+  );
+};
+
+// CategoryFilter.propTypes = {
+//   handleFilterChange: PropTypes.func.isRequired,
+// };
```
Comment (krys2fa) [inline src/components/CategoryFilter.js]: - [x] [REQUIRED] It’s better to delete **commented piece of codes** like these. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by krys2fa — APPROVED** [review]
## Awesome work @AbdelrhmanAmin 👏 
Your project design is consistent with the design given and all styling best practices were adhered to. 💪 

### Status: Changes Required ♻️ 
Just a few more adjustments to complete the project. These are general minimum requirements required of every project. They can be seen in the comments below.


Please request another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer :wave:

Thank You for submitting this Pull Request. I am thrilled to inform you that your project is approved now. :clap:

You have done a great job. I like your project. 

**To highlight** 

- [ ] You have a descriptive Readme. Nice Job. 
- [ ] Your Pull Request is nice. Good Job!
- [ ] You have deployed your website. Nice!!!
- [ ] You are using linters and no error is present. I am happy to see it. 
- [ ] Your project has met the minimum technical requirements too. 



In short, I am very excited to see this project and proud of you. 

**What to do next?** :thinking:

1- Visit https://dashboard.microverse.org/projects and submit your approved project. 
2- Prepare yourself for the next project. 

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**